### DISCUSSION TITLE
Why is the group displayed as 5a93d4b92de0, instead of using the group name?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [hyprwm/Hyprland#7260] [Q&A]
![image](https://github.com/user-attachments/assets/c223f1fa-23bb-4c4c-b088-65b798563b54)
Is it because that group should be unique? But I can't create groups with the same name, then why are there hashs?

**Comment by vaxerski** [comment]
grouped: is a list of window addresses of the windows in a particular group
